[2022-04-03 20:52:32,629] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2022-04-03T20:50:37.648617+00:00 [queued]>
[2022-04-03 20:52:32,663] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2022-04-03T20:50:37.648617+00:00 [queued]>
[2022-04-03 20:52:32,664] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 20:52:32,664] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-04-03 20:52:32,664] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-04-03 20:52:32,679] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2022-04-03T20:50:37.648617+00:00
[2022-04-03 20:52:32,697] {standard_task_runner.py:52} INFO - Started process 6268 to run task
[2022-04-03 20:52:32,721] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2022-04-03T20:50:37.648617+00:00', '--job-id', '47', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpna8wc68v', '--error-file', '/tmp/tmp3hb2mjhe']
[2022-04-03 20:52:32,727] {standard_task_runner.py:77} INFO - Job 47: Subtask task_4
[2022-04-03 20:52:32,801] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2022-04-03T20:50:37.648617+00:00 [running]> on host airflowvm
[2022-04-03 20:52:32,895] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2022-04-03T20:50:37.648617+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-04-03T20:50:37.648617+00:00
[2022-04-03 20:52:32,896] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-04-03 20:52:32,896] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-04-03 20:52:32,902] {subprocess.py:75} INFO - Output:
[2022-04-03 20:52:35,923] {subprocess.py:83} INFO - Command exited with return code 0
[2022-04-03 20:52:35,983] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20220403T205037, start_date=20220403T205232, end_date=20220403T205235
[2022-04-03 20:52:36,037] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-03 20:52:36,071] {local_task_job.py:151} INFO - Task exited with return code 0
